dynawo/sources/Launcher/dynawo.sh

```diff
@@ -122,6 +122,8 @@ while (($#)); do
       ;;
     jobs-help)
       shift
+      echo "Usage: dynawo.sh jobs jobs-file"
```

Comment:
It would be nice to put something around jobs-file so that people understand they have to replace it

Reply:
yes it is better